app/helpers/mailer_helper.rb

```diff
@@ -0,0 +1,11 @@
+module MailerHelper
+  def delete_account_url_for(user)
+    delete_confirmation_user_url host: organic_domain_for(user), token: user.delete_account_token
+  end
+
+  private
+
+  def organic_domain_for(user)
+    Mumukit::Platform.laboratory.organic_domain(user.last_organization)
```

Comment:
Prefer to explicitly obtain organization from where the user clicked delete profile button